2021-08-29 10:37:18,450 INFO    MainThread:44106 [wandb_setup.py:_flush():69] setting env: {}
2021-08-29 10:37:18,450 INFO    MainThread:44106 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-29 10:37:18,450 INFO    MainThread:44106 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/code/image-classification-level1-22/T2037/wandb/run-20210829_103718-3r66x4i5/logs/debug.log
2021-08-29 10:37:18,450 INFO    MainThread:44106 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/code/image-classification-level1-22/T2037/wandb/run-20210829_103718-3r66x4i5/logs/debug-internal.log
2021-08-29 10:37:18,450 INFO    MainThread:44106 [wandb_init.py:init():375] calling init triggers
2021-08-29 10:37:18,450 INFO    MainThread:44106 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-08-29 10:37:18,451 INFO    MainThread:44106 [wandb_init.py:init():424] starting backend
2021-08-29 10:37:18,451 INFO    MainThread:44106 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-29 10:37:18,462 INFO    MainThread:44106 [backend.py:ensure_launched():135] starting backend process...
2021-08-29 10:37:18,470 INFO    MainThread:44106 [backend.py:ensure_launched():139] started backend process with pid: 44129
2021-08-29 10:37:18,485 INFO    MainThread:44106 [wandb_init.py:init():429] backend started and connected
2021-08-29 10:37:18,486 INFO    MainThread:44106 [wandb_init.py:init():477] updated telemetry
2021-08-29 10:37:18,487 INFO    MainThread:44106 [wandb_init.py:init():500] communicating current version
2021-08-29 10:37:19,204 INFO    MainThread:44106 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-29 10:37:19,205 INFO    MainThread:44106 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-29 10:37:19,427 INFO    MainThread:44106 [wandb_init.py:init():540] starting run threads in backend
2021-08-29 10:37:24,432 INFO    MainThread:44106 [wandb_run.py:_console_start():1584] atexit reg
2021-08-29 10:37:24,432 INFO    MainThread:44106 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-08-29 10:37:24,432 INFO    MainThread:44106 [wandb_run.py:_redirect():1463] Redirecting console.
2021-08-29 10:37:24,436 INFO    MainThread:44106 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-29 10:37:24,436 INFO    MainThread:44106 [wandb_init.py:init():565] run started, returning control to user process
2021-08-29 10:37:27,643 INFO    MainThread:44106 [wandb_watch.py:watch():43] Watching
2021-08-29 10:37:27,647 INFO    MainThread:44106 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 1
2021-08-29 10:37:27,648 INFO    MainThread:44106 [wandb_run.py:_restore():1526] restore
2021-08-29 10:37:30,038 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 947
}

2021-08-29 10:37:30,141 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 947
}

2021-08-29 10:37:30,354 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 947
}

2021-08-29 10:37:30,458 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10920
}

2021-08-29 10:37:30,560 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10920
}

2021-08-29 10:37:30,662 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10920
}

2021-08-29 10:37:30,763 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10920
}

2021-08-29 10:37:30,865 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:37:30,967 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:37:31,069 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:37:31,171 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:37:31,273 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:37:31,679 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:37:31,781 INFO    MainThread:44106 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:37:32,901 INFO    MainThread:44106 [wandb_run.py:_show_files():1898] logging synced files
